.github/travis/script.sh

```diff
@@ -22,14 +22,18 @@ export TRAVIS_TAG=${TRAVIS_TAG:-$(git describe --match=v*)}
 BAZEL_CXXOPTS=(--cxxopt=-std=c++17)
 
 case "$MODE" in
-compile-n-test)
-    bazel test --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
-    bazel build -c opt --noshow_progress "${BAZEL_CXXOPTS[@]}" //...
+test)
+    bazel test -c opt //...
+    ;;
+
+compile)
+    bazel build -c opt //...
```

Comment:
You'll want `"${BAZEL_CXXOPTS[@]}"` as c++17 it is required by one of Verible's prerequisites (until we fix it).

Reply:
done